po/pl.po

```diff
@@ -0,0 +1,441 @@
+msgstr "Wyłączone"
+
+#. Translators: the LED has a single, solid color.
+#: piper/ratbagd.py:835 data/ui/LedDialog.ui:66
+msgid "Solid"
+msgstr "Jeden kolor"
+
+#. Translators: the LED is cycling between red, green and blue.
+#: piper/ratbagd.py:837 data/ui/LedDialog.ui:188
+msgid "Cycle"
+msgstr "Zmienny kolor"
+
+#. Translators: the LED's is pulsating a single color on different
+#. brightnesses.
+#: piper/ratbagd.py:840 data/ui/LedDialog.ui:353
+msgid "Breathing"
+msgstr "Pulsujący kolor"
+
+#: piper/window.py:50
+msgid "Cannot connect to ratbagd"
+msgstr "Nie można połączyć z usługą ratbagd"
+
+#: piper/window.py:51
+msgid "Please make sure ratbagd is running"
+msgstr "Proszę się upewnić, że usługa ratbagd jest uruchomiona"
+
+#: piper/window.py:64 piper/window.py:114
+msgid "Cannot find any devices"
+msgstr "Nie można odnaleźć żadnego urządzenia"
+
+#: piper/window.py:65 piper/window.py:115
+msgid "Please make sure your device is supported and plugged in"
+msgstr "Proszę się upewnić, że urządzenie jest obsługiwane i podłączone"
+
+#: piper/window.py:77
+msgid "There are unapplied changes. Are you sure you want to quit?"
+msgstr "Są niezastosowane zmiany. Na pewno zakończyć?"
+
+#. The current device disconnected, which can only happen from the
+#. mouse perspective as we'd otherwise be in the welcome screen with
+#. more than one device remaining. Hence, we display the error
+#. perspective.
+#: piper/window.py:106
+msgid "Your device disconnected!"
+msgstr "Urządzenie zostało rozłączone."
+
+#: piper/window.py:107
+msgid "Please make sure your device is plugged in"
+msgstr "Proszę się upewnić, że urządzenie jest podłączone"
+
+#: piper/window.py:155
+msgid "Cannot display device SVG"
+msgstr "Nie można wyświetlić obrazu SVG urządzenia"
+
+#: piper/window.py:157
+msgid "Unknown exception occurred"
+msgstr "Wystąpił nieznany wyjątek"
+
+#: data/ui/AboutDialog.ui.in:13
+msgid "Visit Piper's website"
+msgstr "Strona programu Piper"
+
+#: data/ui/ButtonDialog.ui:65
+msgid "Search for a button mapping"
+msgstr "Wyszukuje mapowanie przycisku"
+
+#: data/ui/ButtonDialog.ui:91
+msgid "Capture a macro for this button"
+msgstr "Przechwytuje makro dla tego przycisku"
+
+#: data/ui/ButtonDialog.ui:104
+msgid "The currently set macro"
+msgstr "Obecnie ustawione makro"
+
+#: data/ui/ButtonDialog.ui:129
+msgid "Send keystroke"
+msgstr "Wyślij kombinację klawiszy"
+
+#: data/ui/ButtonDialog.ui:169
+msgid "Enter a new key sequence for the macro."
+msgstr "Nowa sekwencja klawiszy dla makra."
+
+#: data/ui/ButtonDialog.ui:221
+msgid "Press Escape to cancel or Return to accept."
+msgstr "Klawisz Esc anuluje, a Enter przyjmie."
+
+#: data/ui/ButtonDialog.ui:252
+msgid "Set the device's handedness."
+msgstr "Lewo- lub praworęczność urządzenia."
+
+#: data/ui/ButtonDialog.ui:270
+msgid ""
+"This allows you to swap the order of the primary mouse buttons. In left-"
+"handedness mode, the physical left mouse button generates a right mouse "
+"click and vice versa."
```

Comment:
Something feels fishy here. Are you sure it’s consistent with the strings below?